elasticdl_preprocessing/layers/round_identity.py

```diff
@@ -0,0 +1,71 @@
+import tensorflow as tf
+from tensorflow.python.ops.ragged import ragged_functional_ops, ragged_tensor
+
+
+class RoundIdentity(tf.keras.layers.Layer):
+    """Implements numeric feature rounding with a max value.
```

Comment:
I think this layer is casting a numeric feature into a categorical feature.

Reply:
How do you define the categorical feature?

Reply:
Maybe cast a numeric feature into a d discretized/categorical value.
IMO, categorical value is discretized and enumerable. After executing this layer, the value become enumerable: [0, 1, 2, 3, ..., MAX_VALUE]

Reply:
Cast a numeric feature into a discretized integer value?

Reply:
discretized integer value?

Reply:
"discrete" is an adj and the code spell checker in VScode hints that "discretized" is an unknown word.